_install-and-configure/os-comp.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Compatible operating systems
+nav_order: 12
+---
+
+OpenSearch and OpenSearch Dashboards are compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [`systemd`](https://en.wikipedia.org/wiki/Systemd), such as Amazon Linux, and Ubuntu Long-Term Support (LTS). While OpenSearch and OpenSearch Dashboards should work on most Linux distributions, we only test a subset. 
+
+## Supported operating systems
+
+The following table lists the operating system versions that we are currently testing on: 
```

Comment:
"that we are currently testing on" => "that we are currently testing" or "that we currently support" (to match the heading)